packages/monitoring-system/data-processor/.gitignore

```diff
@@ -0,0 +1 @@
+package-lock.json
```

Comment:
These were explicitly enabled recently.

Reply:
Ah ok, I thought something changed in the config on my side. I'll remove the ignore.